### PR TITLE
Adds location check by classloader, adds spotless for google-java-format, adds lat/long search for city

### PR DESCRIPTION
**Pull request by jottinger**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -32,6 +32,33 @@
   </dependencies>
   <description>A location service for Java, identify a city, state or country by multiple terms.
   </description>
+  <modules>
+    <module>buildtools</module>
+    <module>location4j</module>
+  </modules>
```
Comment (tomaytotomato) [inline pom.xml:38]: This is a duplicate of the modules on line 90

File: pom.xml [hunk 2]
```diff
@@ -32,6 +32,33 @@
...
+    </sonar.coverage.jacoco.xmlReportPaths>
+    <sonar.language>java</sonar.language>
+    <central-publishing-maven-plugin.version>0.5.0</central-publishing-maven-plugin.version>
+    <maven-gpg-plugin.version>3.2.4</maven-gpg-plugin.version>
+    <maven-deploy-plugin.version>3.1.2</maven-deploy-plugin.version>
+    <spotless.version>2.43.0</spotless.version>
+  </properties>
+
   <developers>
     <developer>
```
Comment (tomaytotomato) [inline pom.xml:63]: Please add yourself to the list of developers if you feel like it.

**Review by tomaytotomato — CHANGES_REQUESTED** [review]
Thanks @jottinger for this awesome PR.

Just a minor issue with maven pom.xml having a duplicate modules tag.

Also please update the readme with your new functionality you added, thanks

File: pom.xml (hunk 1 above)
Comment (jottinger) [reply pom.xml]: Yeah, the, uh, merge didn't work quite as was desired. Fixing.

File: pom.xml (hunk 2 above)
Comment (jottinger) [reply pom.xml]: I'm good. If you want to add me, that's fine, but I don't really have a horse in this particular race.

**Pull request merged**